src/7-to-8/major-changes/remote-owner.rst

```diff
@@ -0,0 +1,46 @@
+Remote Usernames
+================
+
+.. admonition:: does this change affect me?
+   :class: tip
+
+   * If set task's ``[runtime][remote]owner`` configuration.
+   * If you use ``--owner`` on the command line
+
+
+Overview
+--------
+
+If your username differs beteen the :term:`scheduler` host and job hosts, then
+you may have configured Cylc to run jobs under the correct account using
+``[runtime][remote]owner`` or used the ``--owner`` Cylc command line option
+with commands which access remote hosts.
+
+.. _SSH configuration file: https://man.openbsd.org/ssh_config
+
+Cylc no longer supports specifying the username in this way, we suggest
+configuring your remote username using the `SSH configuration file`_ e.g:
+
+.. code-block:: none
+
+   Host MyHost
+     User root
+
+SSH will then automatically use the configured username when connecting to the
+remote machine.
+
+Since Cylc uses SSH and RSync to manage job hosts, the SSH config also configures
+Cylc.
+
+.. warning::
+
+   GNU/BSD RSync is usually configured to use SSH for its transport layer,
+   however, other options, namely RSH may also be available.
+
+   If using the SSH config to set your remote username ensure RSync is configured
+   to use the SSH transport layer, or that the chosen transport layer is itself
+   appropriately configured.
```

Comment:
I'm not convinced we need this warning. rsync has used ssh by default for many years. Cylc assumes the use of ssh for contacting remote hosts so it's reasonable to assume that rsync will also be using ssh.

Reply:
Makes sense, cheers.